### ISSUE TITLE
When converting tensorflow model to tflite model, is there any way to fix the output order during inference using tflite as Facing an issue of output order of tflite inference on meraki custom cv

### ISSUE THREAD

**Issue by RohanEmpire** [tensorflow/tensorflow#61523]
I took a pretrained model (SSD MobileNet 320x320) for object detection from the TensorFlow Zoo and configured/tuned it according to my data. I trained a TensorFlow model which detects 2 labels.

I have used the latest checkpoint to save the model, then froze it, and finally performed TF Lite conversion. I did this because I need to upload the TF Lite model only to a Cisco camera. 

I'm facing an issue with the output order during TF Lite inference, as the output arrays get jumbled /rearranged. I need help on how to convert the TensorFlow model to TF Lite efficiently. My TensorFlow version is 2.10

**Comment by sushreebarsa** [comment]
@RohanEmpire Please have a look at the [model conversion overview](https://www.tensorflow.org/lite/models/convert) of TFlite to know how to convert TF models to TFlite ones. This [guide](https://www.tensorflow.org/lite/inference_with_metadata/lite_support) also describes well on the processing of inputs and output data of Tflite model. Thank you!
